Makefile

```diff
@@ -56,7 +56,7 @@ bootstrap: ## Download and install all project dependencies (+ prep tooling in t
 		tar -xzvf pkger_0.17.0_Linux_x86_64.tar.gz pkger && \
 		mv pkger $(BIN)
 	# install goreleaser
-	curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | sh -s -- -b $(BIN) v0.138.0
+	GOBIN=$(BIN) go install github.com/goreleaser/goreleaser@latest
```

Comment:
we should be pinning to a specific version of goreleaser and not `latest`

Reply:
Whelp, Weston wins that bet 🤣 . I'll wire him £10 and pin it to the latest version, `v1.3.1` 👍